pkgs/applications/misc/activate-linux/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, pkg-config
+, xorg
+, cairo
+}:
+
+stdenv.mkDerivation rec {
+  pname = "activate-linux";
+  version = "unstable-2022-05-22";
+
+  src = fetchFromGitHub {
+    owner = "MrGlockenspiel";
+    repo = pname;
+    rev = "18a6dc9771c568c557569ef680386d5d67f25e96";
+    sha256 = "wYoCyWZqu/jgqAuNYdNr2bjpz4pFRTnAF7qF4BRs9GE=";
+  };
+
+  makeFlags = [ "PREFIX=$(out)/" ];
```

Comment:
You can remove the trailing slash in the prefix since https://github.com/MrGlockenspiel/activate-linux/pull/75 was merged.